scripts/gulp/preset.ts

```diff
@@ -0,0 +1,34 @@
+import { task, parallel } from 'gulp';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as tsPaths from 'tsconfig-paths';
+
+function registerTsPaths() {
+  const { compilerOptions } = JSON.parse(
+    fs.readFileSync(path.resolve(__dirname, '../../tsconfig.base.v0.json'), 'utf-8'),
```

Comment:
using base config for aliases definition.

q: wondering why are those needed for gulp ? @ling1726  @layershifter

Reply:
@Hotell it might not be required, but as I see there is at least one place that directly requires files from `@fluentui/accessibility`.

https://github.com/microsoft/fluentui/blob/5b9b1f47e2dc20ac0479cde325bf1827914977db/scripts/gulp/plugins/gulp-component-menu-behaviors.ts#L86-L87

Removing aliases will probably cause build failures for `@fluentui/docs`, but you can give a try.

Reply:
gotcha, will test it out in follow-up PR. thx!